### PR TITLE
Get rid of papertrail gem

### PR DESCRIPTION
**Pull request by yulgolem**
See #1561

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,14 @@ language: ruby
 cache: bundler
 env:
   - DB=postgresql
-bundler_args: --without development staging production
```
Comment (yulgolem) [inline .travis.yml]: Deprecated in bundler v 2.1+

**Comment by yulgolem** [comment]
Task for converting old history to new takes up all the memory and Linux kernel kills it so it most probably does not run til the end. It does not log much also so it is hard to tell how much got done.

File: .codeclimate.yml [hunk 2]
```diff
@@ -23,8 +26,17 @@ plugins:
   rubocop:
     enabled: true
     channel: rubocop-0-74
+    checks:
+      Rubocop/Metrics/BlockLength:
+        enabled: false
+      Rubocop/Bundler/OrderedGems:
+        enabled: false
```
Comment (karlerikounapuu) [inline .codeclimate.yml:33]: Are these modifications extremely needy? I would rather suppress errors (if urgently needed) in this branch, but keep config the same for global PR's.

File: app/controllers/admin/domain_versions_controller.rb [hunk 3]
```diff
@@ -1,77 +1,69 @@
...
       end
 
-      whereS += "  AND object->>'registrant_id' IN (#{registrants.map { |r| "'#{r.id.to_s}'" }.join ','})" if registrants.present?
-      whereS += "  AND 1=0" if registrants == []
-      whereS += "  AND object->>'registrar_id' IN (#{registrars.map { |r| "'#{r.id.to_s}'" }.join ','})" if registrars.present?
-      whereS += "  AND 1=0" if registrars == []
+      whereS += add_query(contacts: registrants, field: 'registrant_id') if registrants.present?
+      whereS += '  AND 1=0' if registrants == []
+      whereS += add_query(contacts: registrars, field: 'registrar_id') if registrars.present?
+      whereS += '  AND 1=0' if registrars == []
```
Comment (karlerikounapuu) [inline app/controllers/admin/domain_versions_controller.rb:43]: I know that `whereS` was used before as well, but isn't it wiser to convert it to snake_case as per standard?

File: config/environments/test.rb [hunk 4]
```diff
@@ -12,7 +12,7 @@
   # Do not eager load code on boot. This avoids loading your whole application
   # just for the purpose of running a single test. If you are using a tool that
   # preloads Rails for running tests, you may have to set it to true.
-  config.eager_load = false
+  config.eager_load = true
```
Comment (karlerikounapuu) [inline config/environments/test.rb:15]: Just wondering why. Does it improve run time?

File: test/models/domain/domain_version_test.rb [hunk 5]
```diff
@@ -13,30 +13,30 @@ def teardown
...
+  # def test_assigns_updator_to_paper_trail_whodunnit
+  #   PaperTrail.whodunnit = @user.id_role_username
+  #
+  #   assert_difference '@domain.versions.count', 1 do
+  #     @domain.apply_registry_lock
+  #   end
+  #
+  #   assert_equal(@domain.updator, @user)
+  #   assert_equal(@domain.updator_str, @user.id_role_username)
+  # end
```
Comment (karlerikounapuu) [inline test/models/domain/domain_version_test.rb:39]: Is it WIP? If not, I guess we're better off with simply deleting this file as nothing is being tested.

File: .codeclimate.yml (hunk 2 above)
Comment (karlerikounapuu) [reply .codeclimate.yml:33]: Forgot to highlight `sql_injection` property, actually this was my main concern

File: test/models/domain/domain_version_test.rb (hunk 5 above)
Comment (yulgolem) [reply test/models/domain/domain_version_test.rb:39]: Yep, that's a forgotten leftover. Thanks.

File: .codeclimate.yml (hunk 2 above)
Comment (yulgolem) [reply .codeclimate.yml:26]: I'll leave OrderedGems here - we've already got gems in Gemfile grouped by type, not ordered by aplhabet.

File: config/environments/test.rb (hunk 4 above)
Comment (yulgolem) [reply config/environments/test.rb]: Just some leftover. Removed, thanks.

**Comment by vohmar** [comment]
Internal error is returned when trying to access some history. Have not figured out what is the reason here

error from logs:
```
Completed 500 Internal Server Error in 11ms (ActiveRecord: 1.9ms)
ActiveRecord::AssociationNotFoundError (Association named 'item' was not found on Audit::DomainHistory; perhaps you misspelled it?):
app/controllers/admin/domains_controller.rb:60:in `versions'
```

accessing domain hisotry from domain details view. Even newly registered domains return error. For example:

https://st-adm.infra.tld.ee/admin/domains/86980/domain_versions

at the same time the history for the same domain opend from domain history view:

https://st-adm.infra.tld.ee/admin/domain_versions/2017845

most of the domains tried returned internal error of accessed from domain detials' view (admin/domains/id/domain_versions). At the same time most of the histories open if accessed from setting domain hostory view, but still some don't ie (https://st-adm.infra.tld.ee/admin/domain_versions/2017844)

contact verions tend to work / open,

**Comment by yulgolem** [comment]
Closing due to incativity

**Pull request closed without merging**